### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by YingboMa**
This PR cleans up exports and tests.

### CODE REVIEW

File: test/ode_tableau_convergence_tests.jl [hunk 1]
```diff
@@ -15,16 +15,26 @@ Random.seed!(100)
 ## Convergence Testing
 println("Convergence Test on Linear")
 dts = 1 .//2 .^(8:-1:4)
-testTol = 0.3
+testTol = 0.7
```
Comment (ChrisRackauckas) [inline test/ode_tableau_convergence_tests.jl:18]: that's really high

Comment (YingboMa) [reply test/ode_tableau_convergence_tests.jl:18]: Yes, because for high order methods, it could superconvergence on linear problems.

Comment (ChrisRackauckas) [reply test/ode_tableau_convergence_tests.jl:18]: it would be good to not do that on all of them

Comment (YingboMa) [reply test/ode_tableau_convergence_tests.jl:18]: We could check `sim.𝒪est[:l∞] >= tab.order || abs(sim.𝒪est[:l∞] - tab.order) < 0.3` too.

**Pull request merged**